### ISSUE TITLE
about side face

### ISSUE THREAD

**Issue by StawEndl** [ashawkey/RAD-NeRF#4]
hi, bro, used my own training data which contains some side face data to training, then i found some odd output on the result folder.  
when the person show front face, the odd things do not appear, but when person turn face, the odd things appear.
![img_v2_9db5c9ce-eb68-4c28-9fd2-136fa236a97g](https://user-images.githubusercontent.com/55580043/207484259-df035e1f-ef05-4e25-8c38-b55cd876254f.jpg)
![img_v2_e74f33d6-ad88-467e-846d-78085d4d032g](https://user-images.githubusercontent.com/55580043/207484277-f0fe8dc7-34b2-49b1-9e11-df7e71c5d294.jpg)
![img_v2_4c736732-f790-4ab2-9f6e-32a1b783203g](https://user-images.githubusercontent.com/55580043/207484297-05690eac-4430-48ad-a8bb-eeb823747a4a.jpg)
![img_v2_860c6d79-8996-45e5-b58d-9399eecb732g](https://user-images.githubusercontent.com/55580043/207484307-8db66938-5cdc-451d-ac8a-63b935122916.jpg)

are there some wrong in my dataset? but i found nothing wrong on my data image from video. what should i do next?

**Comment by ashawkey** [comment]
@StawEndl Hi, this method doesn't perform well for side face (since it's rarely observed in the monocular training video). However, the shadow-like artifacts on the second example is abnormal. How long is your training video?

**Comment by StawEndl** [comment]
my whole video is 2 min long, this is me part-data-video:

https://user-images.githubusercontent.com/55580043/208333147-420f8958-0c7f-412a-8325-60879d68f0d9.mp4

and this is my output video:

https://user-images.githubusercontent.com/55580043/208333310-30e0954d-1058-44c6-a07c-7f569bf8c8b8.mp4

are there some wrong with my data-video? thanks for your help.

**Comment by StawEndl** [comment]
an other thing, there are obama.json in README, i just use transforms_train.json which is producted at  to inference output. that is OK?

**Comment by StawEndl** [comment]
after training, where is the _eo.pth file? i just get ngp.pth or ngp_xxxx.pth like ngp_ep0080.pth

**Comment by ashawkey** [comment]
@StawEndl The data seems OK, though it's a little short. 
1. Could you check if the semantic segmentation of face is correct? The test video quality is under expectation... 
2. You could also check the GUI and turn the face to side (like obama). To me it seems the depth (z-axis) is not long enough and the face is flat, but I cannot assure.
3. Yes, the obama.json is in fact the same as transforms_train.json.

**Comment by StawEndl** [comment]
> @StawEndl The data seems OK, though it's a little short.
> 
> 1. Could you check if the semantic segmentation of face is correct? The test video quality is under expectation...
> 2. You could also check the GUI and turn the face to side (like obama). To me it seems the depth (z-axis) is not long enough and the face is flat, but I cannot assure.
> 3. Yes, the obama.json is in fact the same as transforms_train.json.

nice, thanks. i use your model as pretrained model, and that's better.
i would like to know, i should run "python main.py data/obama/ --workspace trial_obama/ -O --iters 200000", then run "python main.py data/obama/ --workspace trial_obama/ -O --iters 250000 --finetune_lips", or just run "python main.py data/obama/ --workspace trial_obama/ -O --iters 250000 --finetune_lips"?

**Comment by ashawkey** [comment]
@StawEndl You should run the two commands sequentially.

**Comment by StawEndl** [comment]
> @StawEndl The two commands should be run sequentially, the second command is a finetuning phase for another 50k steps.

yes,thanks. i run that two commands, without pretrained model, inference output video is nice, and there are nothing wrong about the face like that which was described at my first question.

**Comment by ashawkey** [comment]
Of course! I have updated the readme. Thanks for the suggestion!
